modules/sc-mesh-secure-deployment/src/2_0/features/PHY/PHY_CRA_scripts/SP_CRA_v7.py

```diff
@@ -74,11 +107,15 @@ def display_table(self):
         print("+---------------------+---------------+-------------------+---------------------------+")
         print("|        Time         |     Node IP   |    MAC Address    |   Authentication Result   |")
         print("+---------------------+---------------+-------------------+---------------------------+")
```

Comment:
This should be printed if debug flag is enabled:

`if self.debug:
          print()
    `

Reply:
Applied the recommended changes